www/pages/user/[id]/home.js

```diff
@@ -12,25 +12,29 @@ import styles from "../../../styles/pages/user/[id]/home.module.css";
 
 import { notify } from "../../../store/features/snackbar";
 
-const User = (props) => {
-  const [loading, setLoading] = useState(true);
-  const [ownedMeetings, setOwnedMeetings] = useState([]);
-  const [participantMeetings, setParticMeetings] = useState([]);
+const User = ( props ) => {
+
+  const initialFilters = { owners: [], name: '' };
+
+  const [ loading, setLoading ] = useState( true );
+  const [ meetings, setMeetings ] = useState([]);
+  const [ filters, setFilters ] = useState( filtersConstuctor );
```

Comment:
It looks like `filtersConstructor` isn't defined due to the change from `initialFilters`.